pom.xml

```diff
@@ -25,7 +25,7 @@
         <kfs.version>2017-05-11</kfs.version>
         <rice.version>2.6.0</rice.version>
         <cynergy.version>2.6.0</cynergy.version>
-        <cynergy.version.suffix>current</cynergy.version.suffix>
```

Comment:
Please undo this temporary POM change; I think the unit tests have been updated so that this change is no longer needed.